zurb-template/src/data/event.json

```diff
@@ -1,7 +1,7 @@
 [ 
 { 
-"text": "Snap and Kubernetes Integration for better monitoring and scale", 
-"link": "https://medium.com/intel-sdi/how-we-integrated-kubernetes-with-snap-and-come-up-with-kubesnap-466ac25fc0ff#.61rotoqqw" 
+"text": "Snap plugin diagnostics allows for easier debugging", 
+"link": "https://github.com/intelsdi-x/snap/blob/docs/PLUGIN_DIAGNOSTICS.md" 
```

Comment:
The link to PLUGIN_DIAGNOSTICS.md does not work. It seems that you should change the address to https://github.com/intelsdi-x/snap/blob/master/docs/PLUGIN_DIAGNOSTICS.md